clover/netcdf/describe.py

```diff
@@ -64,14 +64,18 @@ def describe(path_or_dataset):
         }
 
         if dtype not in ('str', ):
-            try:
+            if len(variable.shape) > 2:
+                # Avoid loading the entire array into memory by iterating along the first index (usually time)
+                variable_info.update({
+                    'min': min(variable[i, :].min().item() for i in range(variable.shape[0])),

```

Comment:
I keep looking at this, and wondering if there isn't a more efficient way we can do this with `numpy` operations, but the thing I keep bumping into is that in order to release the memory, we basically have to do this using python loops rather than handing it off to `numpy`, right?